src/Utilities/Hooks/useFetchTotalBatched.js

```diff
@@ -35,7 +35,7 @@ const CONCURRENT_REQUESTS = 2;
  *
  */
 const useFetchTotalBatched = (fetchFn, options = {}) => {
-  const { batchSize = DEFAULT_BATCH_SIZE, skip = false } = options;
+  const { batchSize = DEFAULT_BATCH_SIZE, skip = false, meta } = options; // Accept meta in the options
```

Comment:
This doesn't feel right. According to the API documentation the remediations API has exactly the same response format for "list" requests and also has a `meta.total`, so this should work without any change needed.